module/zcommon/zfs_prop.c

```diff
@@ -583,7 +583,7 @@ zfs_prop_init(void)
 	    "ENCROOT");
 	zprop_register_string(ZFS_PROP_KEYLOCATION, "keylocation",
 	    "none", PROP_DEFAULT, ZFS_TYPE_FILESYSTEM | ZFS_TYPE_VOLUME,
-	    "prompt | <file URI>", "KEYLOCATION");
+	    "prompt | <file URI> | <https URL>", "KEYLOCATION");
```

Comment:
```suggestion
	    "prompt | <file URI> | <https URI>", "KEYLOCATION");
```
Would match how this option is described in `zfsprops(8)`.  Either URI or URL would be fine but let's be consistent in the documentation.

Reply:
Good point, reverted zfsprops(8) to use URL too.